python/ray/node.py

```diff
@@ -262,7 +262,7 @@ def start_redis(self):
              redis_shard_ports=self._ray_params.redis_shard_ports,
              num_redis_shards=self._ray_params.num_redis_shards,
              redis_max_clients=self._ray_params.redis_max_clients,
-             redirect_worker_output=self._ray_params.redirect_worker_output,
+             redirect_worker_output=True,
```

Comment:
How about just remove "redirect_worker_output" of `_make_inc_temp`?

Reply:
@suquark I think that's probably a good idea, but the reason I didn't do it here is that I thought we might still need that functionality if starting workers in `tmux`.